src/components/LaunchDetailsPage/LaunchDetailsPage.scss

```diff
@@ -1,3 +1,11 @@
+.launch-details-card {
+  margin-top: 1rem;
+}
+
+.launch-details-grid {
+  min-height: 60vh;
```

Comment:
Why vh is used here instead rem?

Reply:
I try to make the card fill  at least 60% of screen without depending on screen resolution